source/documentation/05-payment-overview.md

```diff
@@ -101,25 +104,72 @@ The user enters their payment details and clicks **Continue**.
+
+- receive a confirmation email (if you have chosen to send these using GOV.UK Pay)
+- be re-directed to your `return_url` page which will then send the user to your payment confirmation page
+
+##### Confirmation email
+
+The user will receive a payment confirmation email containing:
+
+- a payment reference number
+- the date of payment
+- who the payment was to
+- the total payment amount
+
+You can add a custom paragraph to a payment confirmation email at the [Email notifications page](https://selfservice.payments.service.gov.uk/email-notifications) on the GOV.UK Pay admin site. For further customisation, you can visit [GOV.UK Notify](https://www.notifications.service.gov.uk/) to set up custom notifications. It is recommended to disable GOV.UK Pay notifications if you do this.
+
+##### Confirmation page
+
+The confirmation page is hosted by you and should:
+
+- confirm that the payment has been received successfully
+- contain a reference number (make them short and usable)
+- have a clear payment summary, showing the amount and description
+- clearly state what is going to happen next (this will be different for each service)
+- if applicable, let the user know they will receive a receipt email (services can either use GOV.UK Notify to send email payment receipts or ask GOV.UK Pay to do that for them)
 
-If the payment *cannot* be approved, the user is shown an error page with a message describing the reason, for example:
+Users have different ways of recording this confirmation information, including screenshots, prints, pdf receipts to download, and writing down the reference number and other relevant information. Teams building services should be aware of, and cater for, all these behaviours.
+
+>Read more about confirmation pages in the [service manual](https://www.gov.uk/service-manual/design/confirmation-pages).
+
+#### Payment Flow: Payment fails
+
+The payment can fail at any point in the process due to:
+
++ the payment being declined or the user choosing to cancel
++ a technical error
++ the payment being cancelled by your service
+
+If the payment fails, the user will see a GOV.UK Pay error page. This includes a link to return to your service that states _Go back to try the payment again_. When a user returns to your service after a failed payment you should show them a page that offers useful next steps. This page should:
+
+- confirm that the payment failed
+- offer the user a chance to try the payment again either by initiating a new payment with GOV.UK Pay or using another method supported by your service
+- advise the user of an alternative course of action
+
+An example can be seen below:
 
 ![](https://s3-eu-west-1.amazonaws.com/pay-govuk-documentation/flow-payment-declined.png)
 
-The link to try the payment again sends the user to the ``return_url`` you provided in the initial request.
 
-#### Payment flow: after payment
+#### Payment flow: Check the status of a payment
```

Comment:
Maybe we can boost this in the title hierarchy so it comes across as a more important section